src/tree/ResolvableTreeItemBase.ts

```diff
@@ -22,6 +22,11 @@ export abstract class ResolvableTreeItemBase extends AzExtParentTreeItem impleme
     public abstract parent?: AzExtParentTreeItem | undefined;
     public abstract fullId: string;
 
+    // Setting this forces the tree item to always start out with a spinner icon, and have a "Loading..." description
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    private _temporaryDescription: string | undefined = 'Loading...';
```

Comment:
I know this is a hack, but if we don't make the loading state the default, then it results in some flashing.